contrib/epee/include/hex.h

```diff
@@ -67,6 +67,9 @@ namespace epee
       return out;
     }
 
+    //! Write `src` as has to `out`. `out` must be exactly 2x in size.
```

Comment:
Appears to be a typo in the comment (`has` should be `hex`).

Reply:
Yup, force pushed a correction.